src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex

```diff
@@ -0,0 +1,64 @@
+\documentclass[en]{../../../../../../eplexam}
+
+\hypertitle{Stochastic processes}{6}{INMA}{1731}{2019}{June}{Majeure}
+{Alice Borbáth}
+{Luc Vandendorpe}
+
+\section{}
+
+Consider the discrete-time LTI causal system with transfer function
+
+\begin{equation}
+    G(z)=\frac{z}{\left(z-\frac{1-\epsilon}{\sqrt{2}}(1+j)\right)\left(z-\frac{1-\epsilon}{\sqrt{2}}(1-j)\right)}
+\end{equation}
+\indent where $0<\epsilon\ll1$. Let the input X be white noise with variance $\sigma_X^2$.
+\begin{enumerate}
+    \item Compute the power spectral density of the output Y, i.e. $\gamma_Y(z)$.
+    \item Sketch $\gamma_Y(e^{j\Omega})$. In other words, show how the graph of the function $\Omega \mapsto \gamma_Y(e^{j\Omega})$ looks like. Give an approximation of its local extrema (position and value).
+    \item Realize the system by a difference equation.
+    \item What kind of finite-dimensional model do you obtain ?
+\end{enumerate}
+
+\nosolution
+
+\section{}
+Let Z be an observed random vector in $\mathbb{R}^n$ and $g:\mathbb{R}^n\to \mathbb{R}$ an estimation rule for a stochastic estimand $\Theta$ (Bayesian estimation). Suppose that $\mathbb{E}[g(Z)]=\mathbb{E}[\Theta]$ and that we have a function $h:\mathbb{R}^n \to \mathbb{R}$ such that $\mathbb{E}[h(Z)]=m_{h(Z)}$ is known. Consider the new estimation rule \begin{equation}
```

Comment:
```suggestion
Let $Z$ be an observed random vector in $\mathbb{R}^n$ and $g \colon \mathbb{R}^n\to \mathbb{R}$ an estimation rule for a stochastic estimand $\Theta$ (Bayesian estimation). Suppose that $\mathbb{E}[g(Z)]=\mathbb{E}[\Theta]$ and that we have a function $h \colon \mathbb{R}^n \to \mathbb{R}$ such that $\mathbb{E}[h(Z)]=m_{h(Z)}$ is known. Consider the new estimation rule \begin{equation}
```

Reply:
You're making me feel a certain type of way with those `\colon`s...